sass/_theme_badge.sass

```
@@ -64,6 +65,7 @@
     left: auto
     border: none
     max-width: $nav-desktop-width
+    max-height: 100%
```

Did you mean:
```suggestion
    max-height: 90%
```
or does 100% work for you as well?